.github/actions/chaos-kitten/sarif-converter.py

```
@@ -5,127 +5,130 @@
-    
+
     # Summary table
     summary.append("\n| Severity | Count | status |")
     summary.append("|---|---|---|")
-    
+
     icons = {
         "critical": "🔴",
-        "high": "asd", # wait "asd"?? Typo prevention: "🟠"
+        "high": "asd",  # wait "asd"?? Typo prevention: "🟠"
         "high": "🟠",
```

Ruff (0.15.0)

[error] Dictionary key literal `"high"` repeated

(F601)